e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/CreateNodeIntegrationTest.java

```diff
@@ -110,6 +110,8 @@ void testCreateFile()
         Assertions.assertEquals(201, acsResponse.statusCode());
         Assertions.assertNotNull(acsResponse.jsonPath().get("entry.id"));
 
+        pauseExecution(5000);
```

Comment:
I would extract 5000 to a global constant somewhere, since I imagine we could need it for several tests.

Reply:
A better pattern would be to wait for a short time and retry on the first few failures (e.g. check every second and fail after five checks).

Reply:
Maybe you could use here https://github.com/Alfresco/hxinsight-connector/blob/605fae6579d07ed39882ecd41dc3c46f96ee996d/common-test/src/main/java/org/alfresco/hxi_connector/common/test/util/RetryUtils.java with parametrized max attempts or initial delay

Reply:
used retryWithBackoff() method from RetryUtils.java